WalletWasabi.Fluent/Views/Wallets/Home/History/HistoryTable.axaml

```diff
@@ -189,40 +205,60 @@
       <!-- Incoming -->
       <DataGridTemplateColumn Header="Incoming (BTC)"
                               Width="*"
+                              MinWidth="160"
+                              MaxWidth="170"
```

Comment:
I am not sure about this. What if somebody receives `11111111.11111111` length of amount. Will there be enough space?

Reply:
Update MaxWidth's